modules/component/spec/let/let.directive.spec.ts

```diff
@@ -332,6 +409,80 @@ describe('LetDirective', () => {
     });
   });
 
+  describe('when suspense', () => {
+    beforeEach(waitForAsync(setupLetDirectiveTestComponentSuspense));
+
+    it('should not render when first observable is in suspense state', fakeAsync(() => {
+      letDirectiveTestComponent.value$ = of(true).pipe(delay(1000));
+      fixtureLetDirectiveTestComponent.detectChanges();
+      expect(componentNativeElement.textContent).toBe('');
+      tick(1000);
+      fixtureLetDirectiveTestComponent.detectChanges();
+      expect(componentNativeElement.textContent).toBe('true');
+    }));
+
+    it('should render suspense when next observable is in suspense state', fakeAsync(() => {
+      letDirectiveTestComponent.value$ = of(true);
+      fixtureLetDirectiveTestComponent.detectChanges();
+      letDirectiveTestComponent.value$ = of(false).pipe(delay(1000));
+      fixtureLetDirectiveTestComponent.detectChanges();
+      expect(componentNativeElement.textContent).toBe('suspense');
+      tick(1000);
+      fixtureLetDirectiveTestComponent.detectChanges();
+      expect(componentNativeElement.textContent).toBe('false');
+    }));
+  });
+
+  describe('when suspense template is passed', () => {
+    beforeEach(waitForAsync(setupLetDirectiveTestComponentSuspenseTpl));
+
+    it('should render main template when observable emits next event', () => {
+      letDirectiveTestComponent.value$ = new BehaviorSubject('ngrx');
+      fixtureLetDirectiveTestComponent.detectChanges();
+      expect(componentNativeElement.textContent).toBe('ngrx');
+    });
+
+    it('should render main template when observable emits error event', () => {
+      letDirectiveTestComponent.value$ = throwError(() => 'ERROR!');
+      fixtureLetDirectiveTestComponent.detectChanges();
+      expect(componentNativeElement.textContent).toBe('undefined');
+    });
+
+    it('should render main template when observable emits complete event', () => {
+      letDirectiveTestComponent.value$ = EMPTY;
+      fixtureLetDirectiveTestComponent.detectChanges();
+      expect(componentNativeElement.textContent).toBe('undefined');
+    });
+
+    it('should render suspense template when observable does not emit', () => {
+      letDirectiveTestComponent.value$ = NEVER;
+      fixtureLetDirectiveTestComponent.detectChanges();
+      expect(componentNativeElement.textContent).toBe('Loading...');
+    });
+
+    it('should render suspense template when initial observable is in suspense state', fakeAsync(() => {
+      letDirectiveTestComponent.value$ = of('component').pipe(delay(100));
+      fixtureLetDirectiveTestComponent.detectChanges();
+      expect(componentNativeElement.textContent).toBe('Loading...');
+      tick(100);
+      fixtureLetDirectiveTestComponent.detectChanges();
+      expect(componentNativeElement.textContent).toBe('component');
+    }));
+
+    it('should render suspense template when next observable is in suspense state', fakeAsync(() => {
```

Comment:
Thanks for writing this test, because I was thinking what suspense would do if there's a next observable. Maybe that this should also be mentioned in the docs?